testsuite/integration-arquillian/tests/other/console/src/main/resources/themes/qe/login/theme.properties

```diff
@@ -0,0 +1 @@
+parent=base
```

Comment:
I missed that you recommended to "use keycloak theme as base". I think that there will be more tests on themes and IMO it will be easier to track what has been tested if we use base and add configuration with new tests. In case there will be much duplicated configuration we can change it later.

What do you think?